src/sentry/api/endpoints/sentryapp_avatar.py

```diff
@@ -0,0 +1,11 @@
+from sentry.api.bases.avatar import AvatarMixin
+from sentry.api.endpoints.sentry_apps import SentryAppsEndpoint
+from sentry.models import SentryAppAvatar
+
+
+class SentryAppAvatarEndpoint(AvatarMixin, SentryAppsEndpoint):
+    object_type = "sentryapp"
+    model = SentryAppAvatar
+
+    def get_avatar_filename(self, obj):
+        return f"{obj.slug}.svg"
```

Comment:
It looks like this pattern is similar to the `organization_avatar.py` endpoint. There is a comment there coordinating with something in `organization_details.py`

https://github.com/getsentry/sentry/blob/master/src/sentry/api/endpoints/organization_details.py#L410-L416

Should we add that similar code to `sentry_app_details`, or atleast a TODO so we stay consistent?

Reply:
@leeandher I'm still looking into this but I don't actually understand what that block of code you linked is doing at all because you don't save an organization avatar through the organization details endpoint at all. There's a separate save button that hits the organization avatar endpoint, and everything else on the page saves when you edit it and uses the organization details endpoint. Like, I don't think it's needed/used at all 🤔